tests/pipelines/unclip/test_unclip.py

```diff
@@ -419,6 +419,9 @@ def test_save_load_local(self):
     def test_save_load_optional_components(self):
         return super().test_save_load_optional_components()
 
+    def test_float16_inference(self):
+        super().test_float16_inference(expected_max_diff=1.0)
```

Comment:
ERROR: (gcloud.alpha.compute.tpus.queued-resources.create) HttpError accessing <https://tpu.googleapis.com/v2alpha1/projects/huggingface-ml/locations/us-central1-a/queuedResources?alt=json&queuedResourceId=req_1>: response: <{'vary': 'Origin, X-Origin, Referer', 'content-type': 'application/json; charset=UTF-8', 'content-encoding': 'gzip', 'date': 'Tue, 19 Sep 2023 09:12:46 GMT', 'server': 'ESF', 'cache-control': 'private', 'x-xss-protection': '0', 'x-frame-options': 'SAMEORIGIN', 'x-content-type-options': 'nosniff', 'alt-svc': 'h3=":443"; ma=2592000,h3-29=":443"; ma=2592000', 'transfer-encoding': 'chunked', 'status': 429}>, content <{
  "error": {
    "code": 429,
    "message": "Quota limit 'QueuedResourcePerProjectPerRegion,QueuedResourcePerProjectPerZone' has been exceeded. Limit: 20,20 in region us-central1,zone us-central1-a.",
    "status": "RESOURCE_EXHAUSTED",
    "details": [
      {
        "@type": "type.googleapis.com/google.rpc.QuotaFailure",
        "violations": [
          {
            "subject": "project:1049843053967",
            "description": "Quota 'QueuedResourcePerProjectPerRegion' exhausted. Limit 20 in region us-central1"
          },
          {
            "subject": "project:1049843053967",
            "description": "Quota 'QueuedResourcePerProjectPerZone' exhausted. Limit 20 in zone us-central1-a"
          }
        ]
      }
    ]
  }
}
>
This may be due to network connectivity issues. Please check your network settings, and the status of the service you are trying to reach.